.github/workflows/conformance.yml

```diff
@@ -108,7 +125,13 @@ jobs:
           kubectl rollout status -n kube-system ds/antrea-agent --timeout=5m
       - name: Run e2e tests
         run: |
-          ./ci/run-k8s-e2e-tests.sh "--e2e-${{ inputs.test-suite }}"
+          conformance_image_options=()
+          # If the K8s version is 'latest', we need to find the latest conformance image.
+          if [ ${{ inputs.k8s-version == 'latest' }} ]; then
+            latest_tag=$(skopeo list-tags docker://registry.k8s.io/conformance | jq -r '.Tags[]' | sort -V | tail -n 1)
+            conformance_image_options+=(--conformance-image "registry.k8s.io/conformance:${latest_tag}")
```

Comment:
It's not really guaranteed that the conformance image matches the K8s version, right? I wonder if it's really worth to support testing master as it could keep changing at any time.